From ae3990be746a3663e00be6ba14f16c0d009350a9 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:18 -0400
Subject: [redhat] kabi: Adding symbol nla_reserve (lib/nlattr.c)

Message-id: <e8a0bee5d78723d2d81e7c027721b72d49cd5d03.1410510773.git.poros@redhat.com>
Patchwork-id: 95985
O-Subject: [PATCH RHEL7.1 138/233] kabi: Adding symbol nla_reserve (lib/nlattr.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API. To reserve space in the message for
    meta-data (cookie) to be added later in the message processing.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/nla_reserve
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/nla_reserve
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/nla_reserve

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/nla_reserve b/redhat/kabi/kabi-module/kabi_ppc64/nla_reserve
new file mode 100644
index 0000000..8020f20
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/nla_reserve
@@ -0,0 +1,2 @@
+#1-
+0x192d45e4 nla_reserve vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/nla_reserve b/redhat/kabi/kabi-module/kabi_s390x/nla_reserve
new file mode 100644
index 0000000..8f3ee70
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/nla_reserve
@@ -0,0 +1,2 @@
+#1-
+0xdd9ce37b nla_reserve vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/nla_reserve b/redhat/kabi/kabi-module/kabi_x86_64/nla_reserve
new file mode 100644
index 0000000..1dc8ac4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/nla_reserve
@@ -0,0 +1,2 @@
+#1-
+0x96120cdd nla_reserve vmlinux EXPORT_SYMBOL
-- 
1.7.1